Makefile

```diff
@@ -25,9 +29,9 @@ container-image-api-push: container-image-api
 
 container-image-operator:
 	mvn package -am -pl operator -Pcontainer-image -DskipTests -Dquarkus.container-image.image=$(CONSOLE_OPERATOR_IMAGE)
-	operator/bin/generate-catalog.sh $(VERSION)
-	$(CONTAINER_RUNTIME) build -t $(CONSOLE_OPERATOR_BUNDLE_IMAGE) -f operator/target/bundle/console-operator/bundle.Dockerfile
-	$(CONTAINER_RUNTIME) build -t $(CONSOLE_OPERATOR_CATALOG_IMAGE) -f operator/target/catalog.Dockerfile
+	operator/bin/generate-catalog.sh VERSION=$(VERSION) ARCH=$(ARCH) CONSOLE_OPERATOR_IMAGE=$(CONSOLE_OPERATOR_IMAGE) SKIP_RANGE=$(SKIP_RANGE)
```

Comment:
The `integration` github action doesn't use the Makefile, but executes `generate-catalog.sh` directly. I think that's the reason for the CI failure.

Reply:
Thanks, i missed that.